src/components/exchange/ConnectDevice.vue

```diff
@@ -4,7 +4,7 @@
       <v-row justify="center" class="mx-0">
         <h1>Connect your {{ walletName }}</h1>
       </v-row>
-      <v-row justify="center" class="ma-0 mb-10">
+      <v-row v-if="walletName !=='Liquality'" justify="center" class="ma-0 mb-10">
```

Comment:
Can we instead use a method like `isHdWallet` that returns true for ledger and trezor but false for liquality?
Just in case we consider adding more software wallets in the future